src/components/service-viewer/service-viewer.vue

```diff
@@ -84,6 +88,15 @@
       :firstStatus="jobStatus"
       :statusLink="statusLink"
     ></status-card>
+    <div v-if="service.components.joblist">
+    <collapsible-card
+      v-if="service.components.joblist"
+      :expand="0"
+      title="Overview of jobs"
+    >
+      <ListJobs :timeExtent="timeExtent"></ListJobs>
```

Comment:
When you use the component write it as <list-jobs>

Reply:
? Not really clear what you mean with this one

Reply:
as .. list-jobs :)